homework_readme.md

```diff
@@ -0,0 +1,21 @@
+grumble = new GrumbleModel()
+grumble.set({body: "much tired", author: "Doge"})
+grumble1 = new GrumbleModel()
+grumble1.set({body: "such humid", author: "Doge"})
+grumble2 = new GrumbleModel()
+grumble2.set({body: "how about 'no'", author: "Grumpy Cat"})
+grumble3 = new GrumbleModel()
+grumble3.set({body: "stop stealing my bananas", author: "Winston the Cat"})
+
+
+grumble.save().done(function(response){
+  console.log(response)})
+  .fail(function(){
+    console.log("this failed")
+    })
+
+- create a grumblr backbone collection object that contains the 4 grumblr objects above
+
+collection.set([grumble, grumble1, grumble2, grumble3])
```

Comment:
i see that you set the collection, but i think you haven't included the instantiation of the collection. In fact you can just do this in one line.
```
var collection = new GrumblesCollection([grumble, grumble1, grumble2, grumble3])